### PR TITLE
Replace instances of `tokio::spawn` with `spawn_named`

### PR DESCRIPTION
**Pull request by Milo123459**
This PR replaces calls to `tokio::spawn` with an internal function called `spawn_named`. This is to assign names to tasks spawned by the library. For use with https://github.com/tokio-rs/console.

### CODE REVIEW

File: src/model/invite.rs [hunk 1]
```diff
@@ -448,5 +448,5 @@ pub enum InviteTargetType {
 enum_number!(InviteTargetType {
     Normal,
     Stream,
-    EmmbeddedApplication,
+    EmmbeddedApplication
```
Comment (nickelc) [inline src/model/invite.rs]: this change is unrelated
```suggestion
    EmmbeddedApplication,
```

Comment (Milo123459) [reply src/model/invite.rs]: `cargo +nightly fmt` is a devil on Windows.

**Comment by Milo123459** [comment]
Anything need to change?

**Review by arqunis — APPROVED** [review]
